### DISCUSSION TITLE
ICE and IC trains not shown (fixed)

### DISCUSSION THREAD

**Discussion by jntnh** [KristjanESPERANTO/MMM-PublicTransportHafas#23] [Q&A] [answered]
Hey guys, 
I've installed the module and think it's really nicely done and helpful. Unfortunately it doesn't show ICE trains but only regional trains, buses etc. I use the standard 'db' profile. Do you know what the problem is here? 
Thanks for your help!

**Comment by KristjanESPERANTO** [comment]
Hi 🙂 Please show your config of the module. Usually it should show IC and ICE.

**Comment by jntnh** [comment]
Hi :) 
thanks for your reply. It shows ICs but unfortunately no ICEs. Is there anybody having the same issue?
Here's my config: 
```
{
module: "MMM-PublicTransportHafas",
position: "top_left",
config: {
    hafasProfile: "db",
    stationID: "8000128",                  
    stationName: "Göttingen", 
    headerPrefix: "Abfahrt von ",
    excludedTransportationTypes: ["bus", "taxi"], 
    ignoredLines: [], 
    timeToStation: 10,
    timeInFuture: 360, 

    displayLastUpdate: true,         
    maxUnreachableDepartures: 2,     
    maxReachableDepartures: 10,      
    showColoredLineSymbols: true,   
    customLineStyles: "", 
    showOnlyLineNumbers: false,
    showTableHeadersAsSymbols: true,
    useColorForRealtimeInfo: true,
    animationSpeed: 0, 
    tableHeaderOrder: [ "time", "line", "direction", "platform" ],
    }
}
```

**Comment by KristjanESPERANTO** [comment, marked as answer]
I fixed it with commit 9ef4a26a82688b39ec5124cd804c2eb12eb4c2da. Please update the module and check it :slightly_smiling_face:

**Reply by jntnh** [reply]
It worked! Thanks a lot for your fast help and the quick fix 👍
